docs/user-guide/replicasets.md

```diff
@@ -36,6 +36,9 @@ provides declarative updates to pods along with a lot of other useful features.
 Therefore, we recommend using Deployments instead of directly using Replica Sets, unless
 you require custom update orchestration or don't require updates at all.
 
+This actually means that you should never need to write a replica set object:
```

Comment:
I agree with @bgrant0607. Also, I think this is already mentioned in the paragraph above. 

> While replica sets can be used independently, today it's mainly used by Deployments as a mechanism to
> orchestrate pod creation, deletion and updates. When you use Deployments you don't have to worry about
> managing the replica sets that they create. Deployments own and manage their replica sets.

But I agree, it is not as strong as your version here.